cmd/cmp/main.go

```diff
@@ -78,11 +89,13 @@ func Run(_ *cobra.Command, args []string) {
 				log.Info().Str("build_side", buildSideFilePath).Str("probe_side", probeSideFilePath).Msg("diff result written")
 				diffWritten++
 			}
+
 			delete(fileIdMap, fileId)
 		}
 	}
-	log.Info().Int("build_side_count", len(fileIdMap)).Int("probe_side_count", probeSideFileCount).
-		Int("file_compared", fileCompared).Int("diff_written", diffWritten).Send()
+
+	log.Info().Int("build_side_count", buildSideFileCount).Int("probe_side_count", probeSideFileCount).
+		Int("files_with_errors", errorFileCount).Int("file_compared", fileCompared).Int("diff_written", diffWritten).Send()
```

Comment:
You also need to add an exit code for `pbench cmp` so Jenkins can tell whether this comparison succeeded or not.
In Go, you can do `os.Exit(exitCode)`

Reply:
What exit codes should I use?

Reply:
We can just do 0 and 1.
0 = all matched
1 = error or mismatch

For Jenkins jobs there is just pass or fail status, so to Jenkins it is just zero or non-zero exit code.